app/models/person.rb

```diff
@@ -56,8 +56,45 @@ def biography
     details["body"]
   end
 
+  def announcement_items
+    announcements.map do |annoucenment|
+      {
+        link: {
+          text: annoucenment["title"],
+          path: annoucenment["link"],
+        },
+        metadata: {
+          public_timestamp: Date.parse(annoucenment["public_timestamp"]).strftime("%d %B %Y"),
+          content_store_document_type: annoucenment["content_store_document_type"].humanize,
+        },
+      }
+    end
+  end
+
+  def has_announcements?
+    announcements.present?
+  end
+
+  def redirect_to_news_and_communications
```

Comment:
```suggestion
  def link_to_news_and_communications
```

Just because this method isn't actually performing any redirect.

Reply:
Makes sense so I've updated it